$NetBSD: patch-Makefile,v 1.2 2021/09/11 15:40:23 fcambus Exp $

- Remove hardcoded debug flag.
- Remove hardcoded optimizations.
- Do not gzip manual pages.
- Fix manual page installation path.

--- Makefile.orig	2021-09-06 12:06:34.000000000 +0000
+++ Makefile
@@ -10,7 +10,7 @@ GIT_HASH ?= $(shell [ -d .git ] && git r
 
 OS ?= $(shell uname -s)
 
-CPPFLAGS = -g -pthread -std=c++20 -fPIE \
+CPPFLAGS = -pthread -std=c++20 -fPIE \
            -DMOLD_VERSION=\"0.9.5\" \
            -DGIT_HASH=\"$(GIT_HASH)\" \
 	   $(EXTRA_CPPFLAGS)
@@ -31,7 +31,6 @@ TSAN ?= 0
 ifeq ($(DEBUG), 1)
   CPPFLAGS += -O0
 else
-  CPPFLAGS += -O2
 endif
 
 ifeq ($(LTO), 1)
@@ -104,10 +103,8 @@ install: all
 	install -m 644 mold-wrapper.so $(DEST)/lib/mold
 	strip $(DEST)/lib/mold/mold-wrapper.so
 
-	install -m 755 -d $(DEST)/share/man/man1
-	install -m 644 docs/mold.1 $(DEST)/share/man/man1
-	rm -f $(DEST)/share/man/man1/mold.1.gz
-	gzip -9 $(DEST)/share/man/man1/mold.1
+	install -m 755 -d $(DESTDIR)$(MAN_DIR)/man1
+	install -m 644 docs/mold.1 $(DESTDIR)$(MAN_DIR)/man1
 
 uninstall:
 	rm -f $(DEST)/bin/mold
